src/sage/combinat/k_regular_sequence.py

```diff
@@ -918,6 +1028,39 @@ def _n_to_index_(self, n):
         except OverflowError:
             raise ValueError('value {} of index is negative'.format(n)) from None
 
+    @cached_method
+    def one(self):
+        r"""
+        Return the one element of this :class:`kRegularSequenceSpace`,
+        i.e. the unique neutral element for `*` and also
+        the embedding of the unique neutral element for `*` in the
+        coefficient ring.
```

Comment:
I do not understand the second part of the sentence.

Reply:
perhaps
"embedding of the one of the coefficient ring into this :class:`kRegularSequenceSpace`" .

Reply:
4ed1e49e790